cmd/security-secretstore-read/res/configuration.toml

```diff
@@ -0,0 +1,36 @@
+#################################################################################
+# Copyright 2020 Redis Labs.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+#################################################################################
+
+[Writable]
+  LogLevel = "ERROR"
+
+[Service]
+  BootTimeout = 30000
+
+[SecretStore]
+  Host = "edgex-vault"
```

Comment:
@tonyespy Do you like to see docker hostname or `localhost` in the default configuration.toml?